e2e/integration/frontity-01/image.spec.js

```diff
@@ -70,9 +70,8 @@ describe("Image lazy-loading (with native lazy-load)", () => {
 
   it("should render an image without loading if it doesn't have height", () => {
     cy.scrollTo("topLeft");
-    cy.get("img:not([height])")
-      .should("not.have.attr", "loading")
-      .should("not.be.visible");
+    cy.get("img:not([height])").should("not.have.attr", "loading");
+    cy.get("img:not([height])").should("not.be.visible");
```

Comment:
I am not certain why that is the case but this test has been failing when run locally when chaining the `.should()` calls:

```js
// doesn't work
cy.get("img:not([height])")
  .should("not.have.attr", "loading")
  .should("not.be.visible");
```

```js
// this works fine 
cy.get("img:not([height])").should("not.have.attr", "loading")
cy.get("img:not([height])").should("not.be.visible");
```